pkg/storaged/anaconda.jsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React from "react";
+import client from "./client.js";
+
+import { StackItem } from "@patternfly/react-core/dist/esm/layouts/Stack/index.js";
+import { Alert } from "@patternfly/react-core/dist/esm/components/Alert/index.js";
+
+const _ = cockpit.gettext;
+
+export const AnacondaAdvice = () => {
+    if (!client.in_anaconda_mode())
+        return null;
+
+    return (
+        <StackItem>
+            <Alert isInline
+                   variant='info'
+                   title={_("What you need to do")}>
+                <p>Anaconda will tell us here what is wrong with the current config.</p>
+            </Alert>
+        </StackItem>
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231205-084155-675f6df4-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)